### ISSUE TITLE
Electrical Age integration crash

### ISSUE THREAD

**Issue by mpevner** [Tonius/NEI-Integration#110]
With the latest versions of Electrical Age, it is possible to induce a crash viewing recipes. I have traced this down to a spelling issue, being a s->z change.
The needed fix is to simply to change Eln.instance.magnetiserRecipes to Eln.instance.magnetizerRecipes in the RecipeHandlerMagnetizer and to use a newer version of ELN as the dependancy. Or so it seems.

**Comment by jrddunbr** [comment]
This problem is also expanded to some fields being private instead of public now. I've been able to make a build of Electrical Age that only crashes on that spelling correction (so I think)

https://github.com/Electrical-Age/ElectricalAge/issues/849

I'm now working on making an edit or two and perhaps a push request to this repository.

Edit: Can't seem to get an API jar for a newer version to compile :(

**Comment by jrddunbr** [comment]
So I compiled ElectricalAge with some fields public and changed the spelling back to an `s` for that one variable, and it worked again. If we can't get a newer dev jar to change that one error, it may be worth pushing a new PR to Electrical Age. Regardless, in the newest version available on Forge's website, the variables required are private, and thus a PR is required either way on the Electrical Age repository.

**Comment by mpevner** [comment]
If you want to comment further, I've already opened up an issue on ELN's side -- https://github.com/Electrical-Age/ElectricalAge/issues/693. As for renaming to z on their side, I am hesitant as to any second order crashes that might cause (other integrations expecting an s).

**Comment by jrddunbr** [comment]
Indeed. Probably best to just leave Integrations alone and fix it on the Eln side, change it back to an `s`.
Perhaps I should do a PR with the code I have working...

**Comment by mpevner** [comment]
Ah, to clarify, leaving the ELN side alone in case there are other integration mods out there. If Tonius is not responsive here, mayhaps on Curesforge?

**Comment by jrddunbr** [comment]
One way or the other, a change is needed to make some fields public on their codebase. If I could get a API jar compiled, then I would make a PR here and there.

**Comment by mpevner** [comment]
Of relevance then -- https://github.com/Electrical-Age/ElectricalAge/pull/652 ; it's not live Yet, it seems the ELN folks have gone on hiatus of sorts. For the future however...

**Comment by jrddunbr** [comment]
1.10 and 1.12 rewrite in progress. They don't want to focus on new features for 1.7.10.

Honestly, for the best future for the mod. Many people (incl myself) would love a 1.12 version.

**Comment by bloxgate** [comment]
Hi. Author of that API PR here. I believe the only thing holding up its merging is that it needs to be rebased now that Baughn is actually able to review things again.

I should be able to take care of that in a couple days.
